source/use-cases/home-resolver.rst

```diff
@@ -1,6 +1,200 @@
+First off, Unbound supports DNSSEC which, through an authentication chain, verifies that the DNS queries you send get a response from the appropriate server as opposed to anyone who has access to the query.
+Secondly, by using your own resolver you can increase your DNS privacy. Because you're not sending out queries to parties who do the resolving for you (your ISP, Google, Cloudflare, Quad9, etc.), you bypass this middle man. While you still send out (parts of) your query unencrypted, you could configure Unbound to take it a step further, which we'll talk about in an upcoming guide.
+Lastly, when you run your own resolver your DNS cache will be locally in your network. Even though the first time you resolve a domain name may be slightly slower than using your ISP’s resolver, all subsequent queries will likely be much faster.
+
+In this tutorial we'll look at setting up Unbound as a DNS resolver; Firstly for your own machine, and then for your entire network.
+
+
+Setting up Unbound
+------------------
+
+Unbound is a powerful validating, recursive, caching DNS resolver. It’s used by some of the biggest tech companies in the world as well as home users, who use it together with ad blockers and firewalls, or self-run resolvers. Setting it up for your home network can be quite simple as we’ll showcase below.
+
+Setting up your own DNS resolver for your entire home network requires a couple of things. Namely, a recursive DNS resolver (who knew!), and a dedicated machine where the resolver runs, which is always on and accessible to the entire network. This can be as simple as a Raspberry Pi, or any other machine that is always online, connected to your home router.
+
+Because of the variety of machines that Unbound can run on we cannot create a comprehensive tutorial for all possible options. For the extent of this tutorial we will use :command:`Ubuntu 20.04.1 LTS` as a stepping stone which you could use and adapt for other machines.
+
+While you could download the code from Github and build it yourself, getting a copy can be as simple as running:
+
+.. code-block:: bash
+
+	sudo apt-get update
+	sudo apt install unbound -y
+
+This gives you a full, compiled, and running version of Unbound which behaves as a caching recursive DNS resolver out of the box for the local machine. 
+
+.. after it has been written, link to the local-stub to show how to compile and build.
+
+Do note that the current setup is only reachable on this machine.
+
+Testing the server locally
 --------------------------
 
-Using Unbound as the resolver for a home network.
+To verify that the server works correctly it’s a good idea to test it before committing the entire network to it. Luckily we can test this on the machine that you installed Unbound on (locally) and from any other machine (remotely) that will be using the resolver after we expose Unbound to the network.
+
+The command for local testing is:
```

Comment:
I would say testing locally instead of local testing. Also, is it clear enough that this command should be run on the machine running Unbound and not on a client machine? Perhaps reword this as "The command for testing locally on the Unbound machine is:" ?